src/views/networkpolicies/new/components/NetworkPolicyFormIngress.tsx

```diff
@@ -22,7 +22,7 @@ const NetworkPolicyFormIngress: FC<NetworkPolicyFormIngressProps> = ({
 }) => {
   const { t } = useNetworkingTranslation();
 
-  if (!networkPolicy.ingress.denyAll) return null;
+  if (networkPolicy.ingress.denyAll) return null;
```

Comment:
Can you please explain this code change? Thanks! :)

Reply:
Yes sure. If the denyAll is true, we don't want to show the ingress and egress settings as we  are already saying that we are disabling that